lms/templates/courseware/static_tab.html

```diff
@@ -6,6 +6,11 @@
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
 </%block>
+<%block name="js_extra">
+
+<%include file="../discussion/_js_body_dependencies.html" />
```

Comment:
Hm, @andy-armstrong , can you weigh in here? We want to add mathjax to static tabs, but this doesn't seem like the right way to do it.